usaspending_api/common/helpers/text_helpers.py

```diff
@@ -38,3 +41,19 @@ def slugify_text_for_file_names(text, default=None, max_length=None):
         return value[:max_length] if max_length else value
 
     return default
+
+
+def standardize_whitespace(string):
```

Comment:
This is nice, it might be prudent to use this for the existing FABS and FPDS loaders too.

`canonicalize_string()` in  usaspending_api/references/helpers.py